src/test/java/jalau/at18/azul/FloorTest.java

```diff
@@ -11,57 +10,57 @@ public class FloorTest {
     @Test
     public void testAddTilesToFloor() {
         Floor floor = new Floor();
-        floor.add('R');
-        floor.add('R');
-        floor.addTilestoFloor('D', 3);
+        floor.add(Tile.RED);
```

Comment:
The single character was kind of cryptic, using enum instances improve readability and usage of objects, cool!

Reply:
Yes, now that we all have the Tile class we will make the change in all of our classes